Tests/UnitTests/Paywalls/Events/PaywallEventStoreTests.swift

```diff
@@ -0,0 +1,168 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  PaywallEventStoreTests.swift
+//
+//  Created by Nacho Soto on 9/5/23.
+
+import Foundation
+import Nimble
+@testable import RevenueCat
+
+@available(iOS 15.0, tvOS 15.0, macOS 12.0, watchOS 8.0, *)
+class PaywallEventStoreTests: TestCase {
```

Comment:
Very simple store class thanks to `FileHandle` hiding implementation details. And also very simple tests :)